.travis.yml

```diff
@@ -1,4 +1,4 @@
-sudo: false
+sudo: required
```

Comment:
Why has this been changed?

Reply:
without it travis was fail at olm make install - check travis build history.
I can revert but it will brake travis build.

Reply:
I see that now. Don't put sudo in here anyway, this needs fixing by the maintainer.

Reply:
Ok. Now I'll take it away.